aws/types.go

```diff
@@ -3,8 +3,39 @@ package aws
 import (
 	"io"
 	"sync"
+	"time"
 )
 
+// Context is an alias of the Go v1.7 stdlib's context.Context interface.
+// It is represented as a SDK interface to enable you to use the "WithContext"
+// API methods.
```

Comment:
finish sentence, need to mention it lets you use the WithContext methods when using Go 1.6.